README.md

```diff
@@ -491,13 +497,56 @@ manager.getCredentials(new BaseCallback<Credentials, CredentialsManagerException
 ```
 
 
-5. **Clear credentials**
+5. **Clear credentials:**
 When you want to log the user out:
 
 ```java
 manager.clearCredentials();
 ```
 
+
+### Encryption enforced (Min API 21)
+
+This version expands the minimum version and adds encryption to the data storage. In those devices where a Secure LockScreen has been configured it can require the user authentication before letting them obtain the stored credentials. The class is called `SecureCredentialsManager`.
+
+
+#### Usage
+The usage is similar to the previous version, with the slight difference that the manager now requires a valid `Context` as shown below:
+
+```java
+AuthenticationAPIClient authentication = new AuthenticationAPIClient(account);
+Storage storage = new SharedPreferencesStorage(this);
+SecureCredentialsManager manager = new SecureCredentialsManager(this, authentication, storage);
+```
+
+#### Requiring Authentication
+
+You can require the user authentication to obtain credentials. This will make the manager prompt the user with the device's configured LockScreen, which they must pass correctly in order to obtain the credentials. **This feature is only available on devices where the user has setup a secured LockScreen** (PIN, Pattern, Password or Fingerprint).
```

Comment:
Seems wordy, do we need to explain what a user has to do to pass a LockScreen?

Reply:
I think the interesting point here is letting them know that no extra PIN/Pwd needs to bet setup/remembered by them, as we use the one they already use for their phone. And of course, if the phone didn't have any secure lockscreen setup, this won't work.